walkthroughs/week-1-web-development/examples/stanley/pom.xml

```diff
@@ -32,10 +32,10 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>epartakki-sps-summer20</deploy.projectId>
           <deploy.version>1</deploy.version>
         </configuration>
       </plugin>
     </plugins>
   </build>
-</project>
+</project>
```

Comment:
Nit: Please add a blank line at the end of this file. It doesn't really matter except that GitHub adds a big red marker to the end of any file without a blank line.

Separately, I'm not sure you want to update any of the files in `examples/`. For week 1, you probably don't want to touch this file at all. For future weeks, you probably want to copy the example files to another part of your repo instead of editing them in place.